packages/shared/fastfetchbot_shared/services/scrapers/config.py

```
@@ -0,0 +1,134 @@
+# Filesystem environment variables
+TEMP_DIR = env.get("TEMP_DIR", tempfile.gettempdir())
+WORK_DIR = env.get("WORK_DIR", os.getcwd())
+DOWNLOAD_DIR = env.get("DOWNLOAD_DIR", os.path.join(WORK_DIR, "download"))
+DEBUG_MODE = get_env_bool(env, "DEBUG_MODE", False)
+
+# Cookie/config file directory — defaults to <WORK_DIR>/conf but can be overridden
+CONF_DIR = env.get("CONF_DIR", os.path.join(WORK_DIR, "conf"))
+
+# Templates & Jinja2
+templates_directory = os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates")
+JINJA2_ENV = Environment(
```

Ruff (0.15.6)

[error] By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)